apps/music/js/db.js

```diff
@@ -108,6 +108,12 @@ function initDB() {
       PlayerView.stop();
     }
 
+    // Dismiss the keyboard if it's active, by blurring all the input elements.
+    var inputs = document.getElementsByTagName('input');
+    for (var i = 0; i < inputs.length; i++) {
+      inputs[i].blur();
+    }
```

Comment:
You should just be able to do `document.activeElement.blur();` here.

Also, I think it'd make more sense to do this in `App.showOverlay()`, since we should hide the keyboard when *any* overlay is shown.